src/concrete/ml/onnx/onnx_utils.py

```diff
@@ -461,9 +466,16 @@ def execute_onnx_with_numpy(
             for initializer in graph.initializer
         },
     )
+
     for node in graph.node:
         curr_inputs = (node_results[input_name] for input_name in node.input)
         attributes = {attribute.name: get_attribute(attribute) for attribute in node.attribute}
+
+        if node.op_type in SUPPORTED_ROUNDED_OPERATIONS:
+            attributes["lsbs_to_remove"] = (
+                lsbs_to_remove[0] if node.op_type != "Equal" else lsbs_to_remove[1]
```

Comment:
maybe explain why we have a different behavior for "equal" here ?

Reply:
I added a comment

Reply:
not sure why this was resoled, I don't see any comments

Reply:
It's already explained in tree_to_numpy:
![image](https://github.com/zama-ai/concrete-ml/assets/14870263/ffb17ea6-de89-4d6b-aa6a-cf519152997c)

I'll add it here as well.